src/webgpu/shader/execution/expression/call/builtin/texture_utils.ts

```diff
@@ -4470,7 +4470,7 @@ function createTextureCallsRunner<T extends Dimensionality>(
   const samplerType = isCompare ? 'sampler_comparison' : 'sampler';
 
   const renderTarget = t.createTextureTracked({
-    format: resultFormat,
+    format: 'rgba32uint',
```

Comment:
I think result format can be either `rgba32sint` and `rgba32uint`?

Reply:
I just made it always write to `rgba32uint` so the 3 paths are

* `rgba32uint` -> `rgba32uint`
* `rgba32sint` -> `rgba32uint`
* `rgba32float` -> `rgba32uint`